site/content/docs/5.3/components/button-group.md

```diff
@@ -228,9 +226,7 @@ These 2 vertical variants should not be used because they do not respect the Ora
       <li><a class="dropdown-item" href="#">Dropdown link</a></li>
     </ul>
   </div>
-  <button type="button" class="btn btn-secondary">Button</button>
-  <button type="button" class="btn btn-secondary">Button</button>
-  <div class="btn-group" role="group">
+  <div class="btn-group dropstart" role="group">
```

Comment:
**[Bootstrap Side]** The markup isn't the one shown in the doc for the dropstart.